src/libcore/slice/mod.rs

```diff
@@ -292,8 +292,10 @@ impl<T> [T] {
     /// Returns a reference to an element or subslice, without doing bounds
     /// checking.
     ///
-    /// This is generally not recommended, use with caution! For a safe
-    /// alternative see [`get`].
+    /// This is generally not recommended, use with caution!
+    /// Calling this method with an out-of-bounds index is UB even if the resulting
+    /// reference is not used.
+    /// For a safe alternative see [`get`].
     ///
     /// [`get`]: #method.get
```

Comment:
```suggestion
    /// [`get`]: #method.get
    /// [undefined behavior]: https://doc.rust-lang.org/reference/behavior-considered-undefined.html
```

Reply:
I took the liberty to update the reference link to be relative (so it stays on the version of the rust docs one was looking at).